README.md

```diff
@@ -2,11 +2,26 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=develop&project=mosip_pre-registration-ui&metric=alert_status)](https://sonarcloud.io/dashboard?branch=develop&id=mosip_pre-registration-ui)
 
 
-# Pre-Registration Portal
+## Overview
 
-This repository contains UI code for Pre-Registration portal. Refer [here](https://docs.mosip.io/1.2.0/modules/pre-registration/pre-registration-user-guide) for details about the functions that a resident can perform via this portal.
+MOSIP provides a reference implementation of the pre-registration portal that customized as per country needs. This repository contains UI code for Pre-Registration portal. 
 
-## Build &amp; Deployment Guide
+## Features
+The Pre-Registration Portal streamlines the identity registration process by allowing residents to submit demographic details and documents online, reducing wait times and improving data accuracy. 
+1. Login and Pre-Registration
+2. Appointment Booking
+3. Notification and Acknowledgment System
+4. Registration Centers Data Integration and Synchronization
+
+## Local Setup
+The project can be set up in two ways:
+
+1. [Local Setup with Docker (Easy Setup for Demos)](#local-setup-with-docker-easy-setup-for-demos)
```

Comment:
I don't think we have any local-setup-with-docker-easy-setup-for-demos for PreReg